### PR TITLE
Monkey Cubes expanding will give the new monkey an attack log with their creator

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Why: Big Brother must know who spams the monkeys....

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Can this be put at the beginning of the new monkeys attack log or something and we can check it if it attacks people this is going to be a ton of spam otherwise

**Comment by ExcessiveUseOfCobblestone** [comment]
yeah one sec

**Comment by KorPhaeron** [comment]
Cool, thanks

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -185,7 +185,10 @@
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
 	visible_message("<span class='notice'>[src] expands!</span>")
-	new /mob/living/carbon/monkey(get_turf(src))
+	var/mob/spammer = get_mob_by_key(src.fingerprintslast)
+	var/msg = "Spawned via [src], Last attached Mob: [spammer.real_name] ([spammer.key])"
+	var/mob/living/carbon/monkey/bananas = new /mob/living/carbon/monkey(get_turf(src))
```
Comment (lzimann) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: just `... = new(get_turf(src))`, no need to repeat the type

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 2]
```diff
@@ -185,7 +185,10 @@
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
 	visible_message("<span class='notice'>[src] expands!</span>")
-	new /mob/living/carbon/monkey(get_turf(src))
+	var/mob/spammer = get_mob_by_key(src.fingerprintslast)
+	var/msg = "Spawned via [src], Last attached Mob: [spammer.real_name] ([spammer.key])"
```
Comment (lzimann) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: no need to declare this variable, just put direct. Also change the message to `Spawned via [src] at [COORD(src)], Last attached mob: [key_name(spammer)].`

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 3]
```diff
@@ -185,7 +185,9 @@
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
 	visible_message("<span class='notice'>[src] expands!</span>")
-	new /mob/living/carbon/monkey(get_turf(src))
+	var/mob/spammer = get_mob_by_key(src.fingerprintslast)
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: KOOOOOOOOOOOOOORRRRRRRRRRRRRRRRRRRRRR!!!!!!

Comment (KorPhaeron) [reply code/modules/food_and_drinks/food/snacks_meat.dm]: aaaaaaaaaaaaaaaaaaaaaaa

**Pull request merged**